Automattic/wp-calypso
# Login: Copy updates

This pull request:
- Updates the error messages as per #14090
- Changes text to sentence case as per #14041 
- Makes some other copy changes
- Displays a notice to let users know we are sending them a text message

<img width="635" alt="screen shot 2017-05-18 at 18 02 27" src="https://cloud.githubusercontent.com/assets/275961/26214404/34fea310-3bf4-11e7-8ff8-44c3be401fb5.png">

  
#### Testing instructions
  
1. Run `git checkout update/login-errors`
2. Open the [`Login` page](http://calypso.localhost:3000/log-in)
3. Check that the copy matches the suggestions
4. Log in to a 2FA account
5. Check that you see a notice letting you know we are trying to send an SMS
  
#### Reviews
